src/Compilers/CSharp/Portable/Symbols/TypeUnification.cs

```diff
@@ -262,7 +292,7 @@
         /// <summary>
         /// Return true if the given type contains the specified type parameter.
         /// </summary>
-        private static bool Contains(TypeSymbol type, TypeParameterSymbol typeParam)
+        internal static bool Contains(TypeSymbol type, TypeParameterSymbol typeParam)
```

Comment:
>internal static bool Contains(TypeSymbol type, TypeParameterSymbol typeParam) [](http://example.com/codeflow?start=8&length=77)

Is this a dupe of ```TypeSymbolExtensions.ContainsTypeParameter```? Consider removing this helper then.

Reply:
There was a small difference with regards to error types. Updated `VisitType` to visit the containing type of error types.